src/bp/ui-studio/src/web/views/OneFlow/diagram/index.tsx

```diff
@@ -644,37 +656,60 @@ class Diagram extends Component<Props> {
   }
 
   render() {
+    const isQnA = this.props.selectedWorkflow === 'qna'
     return (
-      <MainContent.Wrapper>
-        <WorkflowToolbar />
-        <Fragment>
-          <div
-            id="diagramContainer"
-            ref={ref => (this.diagramContainer = ref)}
-            tabIndex={1}
-            style={{ outline: 'none', width: '100%', height: '100%' }}
-            onContextMenu={this.handleContextMenu}
-            onDrop={this.handleToolDropped}
-            onDragOver={event => event.preventDefault()}
-          >
-            <div className={style.floatingInfo}>{this.renderCatchAllInfo()}</div>
-
-            <DiagramWidget
-              ref={w => (this.diagramWidget = w)}
-              deleteKeys={[]}
+      <Fragment>
+        {isQnA && (
+          <InjectedModuleView
+            key={`${this.props.selectedTopic}`}
+            moduleName="qna"
+            componentName="LiteEditor"
+            contentLang={this.props.contentLang}
+            extraProps={{
+              isLite: true,
+              topicName: this.props.selectedTopic,
+              languages: this.props.languages,
+              defaultLanguage: this.props.defaultLanguage,
+              getQnaCountByTopic: () => {
+                // So it's processed on the next tick, otherwise it won't update with the latest update
+                setTimeout(() => {
+                  this.props.getQnaCountByTopic()
+                })
+              }
+            }}
+          />
+        )}
+        <MainContent.Wrapper className={cx({ [style.hidden]: isQnA })}>
+          <WorkflowToolbar />
+          <Fragment>
+            <div
+              id="diagramContainer"
+              ref={ref => (this.diagramContainer = ref)}
+              tabIndex={1}
+              style={{ outline: 'none', width: '100%', height: 'calc(100% - 28px)' }}
+              onContextMenu={this.handleContextMenu}
+              onDrop={this.handleToolDropped}
+              onDragOver={event => event.preventDefault()}
+            >
+              <div className={style.floatingInfo}>{this.renderCatchAllInfo()}</div>
+
+              <DiagramWidget
+                ref={w => (this.diagramWidget = w)}
+                deleteKeys={[]}
```

Comment:
I imagine the issue with deleting items comes from here ?

Reply:
The issue is with the `withLanguage` wrapper added to the diagram, I fixed the issue in my other branch for the say node as I needed it as well, it'll be fixed once it's merged